config/vendor_api/v1.0.yml

```diff
@@ -422,7 +422,9 @@ components:
       properties:
         application_url:
           type: string
-          description: The URL of this application as seen in the Manage teacher training applications web interface.
+          nullable: true
```

Comment:
we are not making this field nullable yet, as that would be a breaking change, could we please remove that and only mark deprecated? Same in the release notes